api/overview/azure/virtual-network.md

```diff
@@ -26,14 +28,66 @@ Install the [NuGet package](https://www.nuget.org/packages/Microsoft.Azure.Manag
 Install-Package Microsoft.Azure.Management.Network.Fluent
 ```
 
+#### .NET Core CLI
+
 ```bash
 dotnet add package Microsoft.Azure.Management.Network.Fluent
 ```
 
+### Code Example
+This example shows how you can create a virtual network.
+
+```csharp
+/* 
+  Include these "using" directives...
+  
+  using Microsoft.Azure.Management.Network.Fluent;
+  using Microsoft.Azure.Management.Network.Fluent.Models;
+*/
+using (NetworkManagementClient client = new NetworkManagementClient(credentials))
+{
+    // Define VNet
+    VirtualNetworkInner vnet = new VirtualNetworkInner()
+    {
+        Location = "West US",
+        AddressSpace = new AddressSpace()
+        {
+            AddressPrefixes = new List<string>() { "0.0.0.0/16" }
+        },
+
+        DhcpOptions = new DhcpOptions()
+        {
+            DnsServers = new List<string>() { "1.1.1.1", "1.1.2.4" }
+        },
+
+        Subnets = new List<Subnet>()
+        {
+            new Subnet()
+            {
+                Name = subnet1Name,
+                AddressPrefix = "1.0.1.0/24",
+            },
+            new Subnet()
+            {
+                Name = subnet2Name,
+               AddressPrefix = "1.0.2.0/24",
+            }
+        }
+    };
+   await client.VirtualNetworks.CreateOrUpdateAsync(resourceGroupName, vNetName, vnet);

```

Comment:
Small nit, but can you space it one more so it is aligned with the code block above? It appears offset to the left by one character.